patroni/ctl.py

```diff
@@ -637,21 +640,13 @@ def output_members(cluster, name, extended=False, fmt='pretty'):
         elif m.name == cluster.sync.sync_standby:
             role = 'Sync standby'
 
-        host = m.conn_kwargs()['host']
-
-        xlog_location = m.data.get('xlog_location') or 0
-        lag = ''
-        if xlog_location_cluster >= xlog_location:
+        xlog_location = m.data.get('xlog_location')
+        lag = 'unknown'
+        if xlog_location is not None and xlog_location_cluster >= xlog_location:
```

Comment:
On at least one occasion I've seen the second condition (xlog_location_cluster >= xlog_location) as not true consistently for the running test cluster (the cluster has not been receiving any data): https://gist.github.com/alexeyklyukin/6b84df0c129216ba15e43d0421daed20

I cannot reproduce that issue from now on, but I think it's worth complaining about such conditions instead of just hiding them under 'Unknown' (perhaps add it to the service_info line).